generator/C/include_v2.0/mavlink_types.h

```diff
@@ -15,6 +15,10 @@ namespace mavlink {
 // Macro to define packed structures
 #ifdef __GNUC__
   #define MAVPACKED( __Declaration__ ) __Declaration__ __attribute__((packed))
+#elif defined __arm__
```

Comment:
Could you find a tighter constraint on the check here?

Does `__UVISION_VERSION` work?